### PR TITLE
Ticket 9 branch

### PR DESCRIPTION
**Pull request by DanielCouperthwaite**
Hi, I have completed ticket 9. I don't quite have the api for the topics themselves but have requested them for the menu page. instead I have tried to hard code 3 different 'featured articles' from the api so that the pages have something to show. If there is a way to do this otherwise then do please let me know though. Thanks!

### CODE REVIEW

File: Components/ArticleBlockByTopic.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom'
+import { useState } from 'react'
+
+export default function ({currentArticles}) {
+    
+    console.log('insideArticleBlockByTopic', currentArticles)
```
Comment (EddTheDuck) [inline Components/ArticleBlockByTopic.jsx:6]: Please remove console.log

File: src/utils/api.js [hunk 2]
```diff
@@ -15,6 +15,19 @@ export const getTopics = () => {
...
+        if(topic === 'cooking'){
+            return [data.articles[7], data.articles[17], data.articles[18]]
+        } else if (topic === 'football'){
+            return [data.articles[8], data.articles[11], data.articles[16]]
+        } else if (topic === 'coding'){
+            return [data.articles[19], data.articles[21], data.articles[31]]
+        }
+    })
+}
+
```
Comment (EddTheDuck) [inline src/utils/api.js:30]: I appreciate you didn't build the query into your BE, however this approach is very brittle.

Try fetching all articles and using a filter to find the articles with the topic you are looking for.

**Review by EddTheDuck — CHANGES_REQUESTED** [review]
Some comments here. Please address these and resubmit.

File: src/utils/api.js (hunk 2 above)
Comment (DanielCouperthwaite) [reply src/utils/api.js:30]: Okay perfect. I gave that a try yesterday but wasn't sure how to filter each article like using array methods. Should I try that before the map?

Comment (DanielCouperthwaite) [reply src/utils/api.js:25]: Ahh I just managed it, thanks!

**Pull request merged**